src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -339,13 +376,13 @@ TEST_CASE("generate soroban load", "[loadgen][soroban]")
         }
     }
 
-    // Manually override network settings for invoke load gen
-    for (auto node : nodes)
-    {
-        overrideSorobanNetworkConfigForTest(*node);
-        modifySorobanNetworkConfig(*node, [&](SorobanNetworkConfig& cfg) {
+    // Override network settings for invoke load gen
```

Comment:
Should we update this comment? We're not "overriding" configs anymore, as the network just goes through a normal upgrade flow.